homeassistant/components/teslemetry/button.py

```diff
@@ -74,7 +74,7 @@ async def async_setup_entry(
     )
 
 
-class TeslemetryButtonEntity(TeslemetryVehiclePollingEntity, ButtonEntity):
+class TeslemetryButtonEntity(TeslemetryVehicleStreamEntity, ButtonEntity):
```

Comment:
This was a bug that went previously unnoticed, but became apparent due to this change.